src/Symfony/Bridge/Doctrine/Test/DoctrineTestHelper.php

```diff
@@ -25,14 +25,33 @@ class DoctrineTestHelper
     /**
      * Returns an entity manager for testing.
      *
+     * @param \Doctrine\ORM\Configuration|null $config
+     *
      * @return EntityManager
      */
-    public static function createTestEntityManager()
+    public static function createTestEntityManager(\Doctrine\ORM\Configuration $config = null)
```

Comment:
please add a `use` statement instead of using the FQCN (same in the docblock then)